api/grpcserver/interface.go

```diff
@@ -19,6 +20,7 @@ import (
 
 // networkInfo interface.
 type networkInfo interface {
+	peerinfo.PeerInfo
```

Comment:
I think this could be its dedicated method instead of inlined into the underlying object. Wdyt?

```suggestion
	PeerInfo() peerinfo.PeerInfo
```

Reply:
Done